src/main/kotlin/com/vk/kphpstorm/configuration/SetupPluginForProjectDialog.kt

```diff
@@ -32,36 +30,45 @@ class SetupPluginForProjectDialog(private val project: Project) : DialogWrapper(
             else arrayOf(okAction, cancelAction)
 
     override fun createCenterPanel(): JComponent {
```

Comment:
Something is wrong with the dimensions:

<img width="638" alt="Screenshot 2024-02-01 at 16 59 01" src="https://github.com/VKCOM/kphpstorm/assets/6605618/acf09e03-94e1-42a7-86d3-4f923c148e58">